sonaranalyzer-dotnet/rspec/cs/S1301_c#.html

```diff
@@ -22,5 +29,20 @@ <h2>Compliant Solution</h2>
 {
   doSomethingElse();
 }
+
+var foo = variable == 0
+  ? doSomething()
+  : doSomethingElse();
+
+</pre>
+<h2>Exceptions</h2>
```

Comment:
Adding an exception makes the rule description more complex. How about simplifying the description above so that we don't need to add the exception?
Example
```
<p><code>switch</code> statements and expressions are useful when there are many different cases depending on the value of the same expression.</p>
<p>When a switch statement is simple enough to be replaced with a single `if`, `if .. else` or ternary operation, one of these more readable forms should be used instead.</p>
```

Reply:
thanks I updated https://jira.sonarsource.com/browse/RSPEC-2828